op-e2e/faultproofs/alphabet_test.go

```diff
@@ -1,173 +0,0 @@
-package faultproofs
-
-import (
-	"context"
-	"testing"
-	"time"
-
-	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
-	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/challenger"
-	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/disputegame"
-	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
-	"github.com/stretchr/testify/require"
-)
-
-func TestChallengerCompleteDisputeGame(t *testing.T) {
```

Comment:
We haven't actually converted these over to output_alphabet tests yet. The cannon ones are done but not these exhaustive cases (which use the alphabet VM to minimise the game depth).

Reply:
Would you be open to adding them back in a follow up? Saw that these weren't covered, though they are the last thing blocking this cleanup. The test cases need to be redone for the new test VM format as well.

Reply:
Yeah probably could do that. I can probably take a look at them today anyway.